manifests/deployment-config.yaml

```diff
@@ -0,0 +1,42 @@
+---
+kind: DeploymentConfig
+apiVersion: apps.openshift.io/v1
+metadata:
+  name: peribolos-as-services
+spec:
+  test: false
```

Comment:
Don't we also need to define a `ServiceAccount` and add a `serviceAccountName` field here so that we have access to secret creation?

Reply:
Good catch, yes, we need a service account with a role granting `get,list,create,update` on `secrets` and `create` on `taskruns`. Let's branch that into a separate issue, so somebody can work on it independently. :slightly_smiling_face:

Reply:
ack, i have attached the serviceaccount.
created a separate issue out of it.
thanks